pkgs/development/compilers/gcc/patches/default.nix

```diff
@@ -154,29 +163,30 @@ in
       hash = "sha256-E4zEKm4tMhovOJKc1/FXZCLQvA+Jt5SC0O2C6SEvZjI=";
     })
     ./14/fixup-gcc-14-darwin-aarch64-support.patch
+    ./cfi_startproc-reorder-label-2.diff
```

Comment:
```suggestion
```

there shouldn't be a `.cfi_startproc-reorderxxxx` patch for gcc-14. it is should be part of Iains patch

Reply:
Fixed.